### PR TITLE
Fix builtin_fp16.h path according to: https://discuss.tvm.apache.org/…

### PR DESCRIPTION
**Pull request by LightricksNatanKaminsky**
…t/builtin-fp16-h-file-not-found/10746

Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Review by tqchen — APPROVED** [review]

File: src/runtime/contrib/sort/sort.cc
Comment (tqchen) [inline src/runtime/contrib/sort/sort.cc:24]: Thank you, please fix the lint error by changing the order of include

**Review by tqchen — CHANGES_REQUESTED** [review]

**Comment by junrushao1994** [comment]
I just re-triggered the CI. One thing I am not sure about: why we use relative path instead of `-I` instead?

**Comment by junrushao1994** [comment]
This CI issue is addressed by this commit: https://github.com/apache/tvm/commit/f5661f4e78cae5f62bc172559040576cd2fd20f2. Just rebased the PR to the HEAD of mainline

**Comment by junrushao1994** [comment]
I’m not super sure about this patch, so would love to wait for more response before getting it merged :-)

**Review by tqchen — APPROVED** [review]

**Comment by junrushao1994** [comment]
okay TQ has an explanation here: https://github.com/apache/tvm/pull/8719

**Pull request merged**
